Tests/RestAPI/OktaAPIRequestTests.swift

```diff
@@ -135,8 +135,51 @@ class OktaAPIRequestTests : XCTestCase {
         let httpResponse = HTTPURLResponse(url: url, statusCode: 400, httpVersion: nil, headerFields: nil)!
         let data = "{\"errorCode\":\"\(errorCode)\"}".data(using: .utf8)!
         
-        req.handleResponse(data: data, response: httpResponse)
+        req.handleResponse(data: data, response: httpResponse, error: nil)
         
         wait(for: [exp], timeout: 1.0)
     }
+    
+    func testInjectHttpClientSuccessResponse() {
+        
+        let httpResponse = HTTPURLResponse(url: url, statusCode: 200, httpVersion: nil, headerFields: nil)!
+        let data = "{\"status\":\"SUCCESS\"}".data(using: .utf8)!
+        
+        let mock = OktaAuthHTTPClientMock(data: data, httpResponse: httpResponse, error: nil)
+        let status = AuthStatus.success
+        let exp = XCTestExpectation(description: "Success result")
+        let req = OktaAPIRequest(baseURL: url, urlSession: URLSession.shared, httpClient: mock) { (req, res) in
+            if case .success(let response) = res, response.status == status {
+                exp.fulfill()
+            } else {
+                XCTFail()
+            }
+        }
+        
+        req.handleResponse(data: data, response: httpResponse, error: nil)
```

Comment:
Not sure why you calling `handleResponse`, you have to call `run` and make sure that delegated method is called